projects/packages/search/src/widgets/js/search-widget-admin.js

```diff
@@ -1,11 +1,10 @@
 /* globals jetpack_search_filter_admin, jQuery, analytics */
 
-( function( $, args ) {
-	var defaultFilterCount = ( 'undefined' !== typeof args && args.defaultFilterCount ) ?
-		args.defaultFilterCount :
-		5; // Just in case we couldn't find the defaultFiltercount arg
+( function ( $, args ) {
```

Comment:
There are a lot of changes in this file to make the linter happy, but I can't see what we're changing. Is `getJetpackData` used in here? If not, then perhaps we can remove this and fix the code style issues in a separate PR.

Reply:
So this file start showing up in the changes when I merged the master in my branch yesterday. It doesn't have any changes related to getJetpackData. Should I remove it from here? Not sure why it started showing in the modified file list in the first place :o

Reply:
Looks like I accepted the wrong change while resolving conflicts. [This file](https://github.com/Automattic/jetpack/blob/master/projects/packages/search/src/widgets/js/search-widget-admin.js) has the changes which I am trying to remove in this PR. I will reset this change in my PR and commit again. @pablinos